### ISSUE TITLE
Be able to toggle writers on and off

### ISSUE THREAD

**Issue by maphysics** [datacleaner/DataCleaner#471]
It would be nice to be able to toggled a writer on and off.  That way if a user is running a job as a template, writers do not need to be deleted and recreated once the new data source is validated.

**Comment by kaspersorensen** [comment]
Good idea. Thinking a bit further on this - I guess it could be interesting to allow _any_ component to be disabled/enabled. That way you could quite easily turn off part of a job if you are trying to figure out what's going on another part or so.

**Comment by maphysics** [comment]
That's a good idea.  It would make debugging a job a lot easier.
